todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/notification/NotificationPublisher.java

```diff
@@ -0,0 +1,20 @@
+package com.nearsoft.androidschool.todoapp.activities.notification;
+
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class NotificationPublisher extends BroadcastReceiver{
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        Notification notification = intent.getParcelableExtra("notification");
```

Comment:
use a string constant here NOTIFICATION_EXTRA = "notification", instead of hardcoded string